x-pack/plugins/rule_registry/server/alert_data_client/alerts_client.ts

```diff
@@ -128,18 +138,19 @@ interface GetAlertSummaryParams {
   fixedInterval?: string;
 }
 
-interface SingleSearchAfterAndAudit {
-  id?: string | null | undefined;
-  query?: string | object | undefined;
-  aggs?: Record<string, any> | undefined;
+interface SearchAlertsParams {
+  id?: string | null;
+  query?: string | object;
+  aggs?: Record<string, any>;
   index?: string;
-  _source?: string[] | undefined;
+  _source?: string[] | false;
```

Comment:
curious about this type, why can it be `false`?

Reply:
To disable the `_source` field ([docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-fields.html#search-fields-request))